### PR TITLE
CDM-648 Add options proxy_connect_timeout and proxy_read_timeout

### PR DESCRIPTION
**Pull request by MarianaStrix**

### CODE REVIEW

File: gcore/resource_gcore_cdn_options.go [hunk 1]
```diff
@@ -496,6 +496,56 @@ var (
...
+			Computed:    true,
+			Description: "The time limit for receiving a partial response from the origin. If no response is received within this time, the connection will be closed.",
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"enabled": {
+						Type:     schema.TypeBool,
+						Optional: true,
+						Default:  true,
+					},
+					"values": {
```
Comment (freenoth) [inline gcore/resource_gcore_cdn_options.go:537]: `values`? As I know it should be a single string value?

Comment (freenoth) [reply gcore/resource_gcore_cdn_options.go:537]: the same for `proxy_connect_timeout`. bad copy paste

**Review by freenoth — CHANGES_REQUESTED** [review]

File: gcore/resource_gcore_cdn_options.go (hunk 1 above)
Comment (MarianaStrix) [reply gcore/resource_gcore_cdn_options.go:532]: Thanks, I got confused with another option

File: docs/resources/cdn_resource.md [hunk 2]
```diff
@@ -415,6 +417,27 @@ Optional:
 
 - `enabled` (Boolean)
 
+<a id="nestedblock--options--proxy_connect_timeout"></a>
+### Nested Schema for `options.proxy_connect_timeout`
+
+Required:
+
+- `value` (Map of String) Specify time in seconds ('1s', '5s' for example).
```
Comment (freenoth) [inline docs/resources/cdn_resource.md:425]: @MarianaStrix please also regenerate docs with the correct types, it shouldn't be a `Map of String`

**Review by freenoth — CHANGES_REQUESTED** [review]

**Review by freenoth — APPROVED** [review]

**Pull request merged**